### PR TITLE
Arabic language

### PR DESCRIPTION
**Pull request by HouariZegai**
I translated the RAEDME page to Arabic language.

-----
[View rendered README.ar-DZ.md](https://github.com/HouariZegai/ant-design-pro/blob/master/README.ar-DZ.md)

### CODE REVIEW

File: README.ar-DZ.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+- :bulb: **تايب سكريبت**: جافا سكريبت لتطوير التطبيقات
+- :scroll: **الكتل**: بناء الصفحة مع قالب كتلة
+- :gem: **تصميم أنيق**: يتبع [مواصفات تصميم النمل](http://ant.design/)
+- :triangular_ruler: **القوالب المشتركة**: قوالب نموذجية لتطبيقات المؤسسة
+- :rocket: **حالة التطور الفني**: أحدث حزمة تطوير لـ React/umi/dva/antd
+- :iphone: **متجاوب**: مصممة لأحجام الشاشات المتغيرة
+- :art: **تصميم**: تصميم قابل للتخصيص مع تكوين بسيط
+- :globe_with_meridians: **عالمي**: تم بناؤه بواسطة i18n solution
+- :gear: **أفضل الممارسات**: سير عمل قوي لجعل الكود سليم
```
Comment (afc163) [inline README.ar-DZ.md:43]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+![](https://user-images.githubusercontent.com/8186664/44953195-581e3d80-aec4-11e8-8dcb-54b9db38ec11.png)
+
+</div>
+
+- معاينة: http://preview.pro.ant.design
+- الصفحة الرئيسية: http://pro.ant.design
+- توثيق: http://pro.ant.design/docs/getting-started
+- سجل التغيير: http://pro.ant.design/docs/changelog
+- الأسئلة الشائعة: http://pro.ant.design/docs/faq
```
Comment (afc163) [inline README.ar-DZ.md:22]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+- مرآة الموقع بالصينية: http://ant-design-pro.gitee.io
+
+## 4.0 صدر الآن! 🎉🎉🎉
+
+[الإعلان عن Ant Design Pro 4.0.0](https://medium.com/ant-design/ant-design-pro-v4-is-here-6f23098ae9d9)
+
+## توظيف الترجمة :loudspeaker:
+
+نحن نحتاج مساعدتك: https://github.com/ant-design/ant-design-pro/issues/120
```
Comment (afc163) [inline README.ar-DZ.md:31]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+![](https://user-images.githubusercontent.com/8186664/44953195-581e3d80-aec4-11e8-8dcb-54b9db38ec11.png)
+
+</div>
+
+- معاينة: http://preview.pro.ant.design
+- الصفحة الرئيسية: http://pro.ant.design
+- توثيق: http://pro.ant.design/docs/getting-started
```
Comment (afc163) [inline README.ar-DZ.md:20]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+| --- | --- | --- | --- | --- |
+| IE11, Edge | last 2 versions | last 2 versions | last 2 versions | last 2 versions |
+
+## المساهمة
+
+نرحب بأي نوع من المساهمات ، وإليك بعض الأمثلة عن كيفية المساهمة في هذا المشروع:
+
+- استخدم Ant Design Pro في عملك اليومي.
+- إرسال [القضايا](http://github.com/ant-design/ant-design-pro/issues) للإبلاغ عن مشكل أو لطرح أسئلة
```
Comment (afc163) [inline README.ar-DZ.md:132]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+![](https://user-images.githubusercontent.com/8186664/44953195-581e3d80-aec4-11e8-8dcb-54b9db38ec11.png)
+
+</div>
+
+- معاينة: http://preview.pro.ant.design
+- الصفحة الرئيسية: http://pro.ant.design
+- توثيق: http://pro.ant.design/docs/getting-started
+- سجل التغيير: http://pro.ant.design/docs/changelog
+- الأسئلة الشائعة: http://pro.ant.design/docs/faq
+- مرآة الموقع بالصينية: http://ant-design-pro.gitee.io
```
Comment (afc163) [inline README.ar-DZ.md:23]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+| --- | --- | --- | --- | --- |
+| IE11, Edge | last 2 versions | last 2 versions | last 2 versions | last 2 versions |
+
+## المساهمة
+
+نرحب بأي نوع من المساهمات ، وإليك بعض الأمثلة عن كيفية المساهمة في هذا المشروع:
+
+- استخدم Ant Design Pro في عملك اليومي.
```
Comment (afc163) [inline README.ar-DZ.md:131]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+- إرسال [القضايا](http://github.com/ant-design/ant-design-pro/issues) للإبلاغ عن مشكل أو لطرح أسئلة
+- اقترح [طلبات السحب](http://github.com/ant-design/ant-design-pro/pulls) لتحسين الكود الخاص بنا.
```
Comment (afc163) [inline README.ar-DZ.md:133]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+
+![](https://user-images.githubusercontent.com/8186664/44953195-581e3d80-aec4-11e8-8dcb-54b9db38ec11.png)
+
+</div>
+
+- معاينة: http://preview.pro.ant.design
+- الصفحة الرئيسية: http://pro.ant.design
+- توثيق: http://pro.ant.design/docs/getting-started
+- سجل التغيير: http://pro.ant.design/docs/changelog
```
Comment (afc163) [inline README.ar-DZ.md:21]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+
+![](https://user-images.githubusercontent.com/8186664/44953195-581e3d80-aec4-11e8-8dcb-54b9db38ec11.png)
+
+</div>
+
+- معاينة: http://preview.pro.ant.design
```
Comment (afc163) [inline README.ar-DZ.md:18]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md (hunk 6 above)
Comment (afc163) [inline README.ar-DZ.md:23]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+
+نحن نحتاج مساعدتك: https://github.com/ant-design/ant-design-pro/issues/120
+
+## الميزات
+
+- :bulb: **تايب سكريبت**: جافا سكريبت لتطوير التطبيقات
+- :scroll: **الكتل**: بناء الصفحة مع قالب كتلة
+- :gem: **تصميم أنيق**: يتبع [مواصفات تصميم النمل](http://ant.design/)
+- :triangular_ruler: **القوالب المشتركة**: قوالب نموذجية لتطبيقات المؤسسة
+- :rocket: **حالة التطور الفني**: أحدث حزمة تطوير لـ React/umi/dva/antd
```
Comment (afc163) [inline README.ar-DZ.md:39]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+
+![](https://user-images.githubusercontent.com/8186664/44953195-581e3d80-aec4-11e8-8dcb-54b9db38ec11.png)
+
+</div>
+
+- معاينة: http://preview.pro.ant.design
+- الصفحة الرئيسية: http://pro.ant.design
```
Comment (afc163) [inline README.ar-DZ.md:19]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

Comment (afc163) [inline README.ar-DZ.md:19]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+[الإعلان عن Ant Design Pro 4.0.0](https://medium.com/ant-design/ant-design-pro-v4-is-here-6f23098ae9d9)
+
+## توظيف الترجمة :loudspeaker:
+
+نحن نحتاج مساعدتك: https://github.com/ant-design/ant-design-pro/issues/120
+
+## الميزات
+
+- :bulb: **تايب سكريبت**: جافا سكريبت لتطوير التطبيقات
+- :scroll: **الكتل**: بناء الصفحة مع قالب كتلة
```
Comment (afc163) [inline README.ar-DZ.md:36]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 14]
```diff
@@ -0,0 +1,135 @@
...
+- توثيق: http://pro.ant.design/docs/getting-started
+- سجل التغيير: http://pro.ant.design/docs/changelog
+- الأسئلة الشائعة: http://pro.ant.design/docs/faq
+- مرآة الموقع بالصينية: http://ant-design-pro.gitee.io
+
+## 4.0 صدر الآن! 🎉🎉🎉
+
+[الإعلان عن Ant Design Pro 4.0.0](https://medium.com/ant-design/ant-design-pro-v4-is-here-6f23098ae9d9)
+
+## توظيف الترجمة :loudspeaker:
```
Comment (afc163) [inline README.ar-DZ.md:29]: Codacy found an issue: [[no-heading-punctuation] Don’t add a trailing `:` to headings](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 15]
```diff
@@ -0,0 +1,135 @@
...
+
+نحن نحتاج مساعدتك: https://github.com/ant-design/ant-design-pro/issues/120
+
+## الميزات
+
+- :bulb: **تايب سكريبت**: جافا سكريبت لتطوير التطبيقات
+- :scroll: **الكتل**: بناء الصفحة مع قالب كتلة
+- :gem: **تصميم أنيق**: يتبع [مواصفات تصميم النمل](http://ant.design/)
```
Comment (afc163) [inline README.ar-DZ.md:37]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md (hunk 4 above)
Comment (afc163) [inline README.ar-DZ.md:20]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 16]
```diff
@@ -0,0 +1,135 @@
...
+- :iphone: **متجاوب**: مصممة لأحجام الشاشات المتغيرة
+- :art: **تصميم**: تصميم قابل للتخصيص مع تكوين بسيط
+- :globe_with_meridians: **عالمي**: تم بناؤه بواسطة i18n solution
+- :gear: **أفضل الممارسات**: سير عمل قوي لجعل الكود سليم
+- :1234: **تطوير وهمي**: سهولة استخدام حل التطوير الوهمي
+- :white_check_mark: **اختبار واجهة المستخدم**: يمكنك الطيران بأمان مع اختبارات الوحدة و e2e
+
+## القوالب
+
+```
```
Comment (afc163) [inline README.ar-DZ.md:49]: Codacy found an issue: [[fenced-code-flag] Missing code language flag](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 17]
```diff
@@ -0,0 +1,135 @@
...
+نحن نحتاج مساعدتك: https://github.com/ant-design/ant-design-pro/issues/120
+
+## الميزات
+
+- :bulb: **تايب سكريبت**: جافا سكريبت لتطوير التطبيقات
+- :scroll: **الكتل**: بناء الصفحة مع قالب كتلة
+- :gem: **تصميم أنيق**: يتبع [مواصفات تصميم النمل](http://ant.design/)
+- :triangular_ruler: **القوالب المشتركة**: قوالب نموذجية لتطبيقات المؤسسة
+- :rocket: **حالة التطور الفني**: أحدث حزمة تطوير لـ React/umi/dva/antd
+- :iphone: **متجاوب**: مصممة لأحجام الشاشات المتغيرة
```
Comment (afc163) [inline README.ar-DZ.md:40]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md (hunk 2 above)
Comment (afc163) [inline README.ar-DZ.md:22]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 18]
```diff
@@ -0,0 +1,135 @@
...
+- :art: **تصميم**: تصميم قابل للتخصيص مع تكوين بسيط
+- :globe_with_meridians: **عالمي**: تم بناؤه بواسطة i18n solution
+- :gear: **أفضل الممارسات**: سير عمل قوي لجعل الكود سليم
+- :1234: **تطوير وهمي**: سهولة استخدام حل التطوير الوهمي
```
Comment (afc163) [inline README.ar-DZ.md:44]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 19]
```diff
@@ -0,0 +1,135 @@
...
+## الميزات
+
+- :bulb: **تايب سكريبت**: جافا سكريبت لتطوير التطبيقات
+- :scroll: **الكتل**: بناء الصفحة مع قالب كتلة
+- :gem: **تصميم أنيق**: يتبع [مواصفات تصميم النمل](http://ant.design/)
+- :triangular_ruler: **القوالب المشتركة**: قوالب نموذجية لتطبيقات المؤسسة
+- :rocket: **حالة التطور الفني**: أحدث حزمة تطوير لـ React/umi/dva/antd
+- :iphone: **متجاوب**: مصممة لأحجام الشاشات المتغيرة
+- :art: **تصميم**: تصميم قابل للتخصيص مع تكوين بسيط
+- :globe_with_meridians: **عالمي**: تم بناؤه بواسطة i18n solution
```
Comment (afc163) [inline README.ar-DZ.md:42]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md (hunk 10 above)
Comment (afc163) [inline README.ar-DZ.md:18]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 20]
```diff
@@ -0,0 +1,135 @@
...
+
+[الإعلان عن Ant Design Pro 4.0.0](https://medium.com/ant-design/ant-design-pro-v4-is-here-6f23098ae9d9)
+
+## توظيف الترجمة :loudspeaker:
+
+نحن نحتاج مساعدتك: https://github.com/ant-design/ant-design-pro/issues/120
+
+## الميزات
+
+- :bulb: **تايب سكريبت**: جافا سكريبت لتطوير التطبيقات
```
Comment (afc163) [inline README.ar-DZ.md:35]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

**Comment by chenshuai2144** [comment]
prettier your code

File: README.ar-DZ.md [hunk 21]
```diff
@@ -0,0 +1,135 @@
...
+  - مراقب
+  - مساحة العمل
+- نموذج
+  - النموذج الأساسي
+  - نموذج الخطوة
+  - نموذج متقدم
+- القائمة
+  - الجدول القياسي
+  - القائمة القياسية
+  - قائمة البطاقات
```
Comment (afc163) [inline README.ar-DZ.md:61]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 22]
```diff
@@ -0,0 +1,135 @@
...
+- الصفحة الرئيسية: http://pro.ant.design
+- توثيق: http://pro.ant.design/docs/getting-started
+- سجل التغيير: http://pro.ant.design/docs/changelog
+- الأسئلة الشائعة: http://pro.ant.design/docs/faq
+- مرآة الموقع بالصينية: http://ant-design-pro.gitee.io
+
+## 4.0 صدر الآن! 🎉🎉🎉
+
+[الإعلان عن Ant Design Pro 4.0.0](https://medium.com/ant-design/ant-design-pro-v4-is-here-6f23098ae9d9)
+
```
Comment (afc163) [inline README.ar-DZ.md:28]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 23]
```diff
@@ -0,0 +1,135 @@
...
+```
+- لوحة القيادة
+  - تحليلي
+  - مراقب
+  - مساحة العمل
+- نموذج
+  - النموذج الأساسي
+  - نموذج الخطوة
+  - نموذج متقدم
+- القائمة
```
Comment (afc163) [inline README.ar-DZ.md:58]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 24]
```diff
@@ -0,0 +1,135 @@
...
+  - الجدول القياسي
+  - القائمة القياسية
+  - قائمة البطاقات
+  - قائمة البحث (Project/Applications/Article)
+- الملف الشخصي
+  - ملف شخصي بسيط
+  - ملف شخصي متقدم
+- الحساب
+  - مركز الحساب
```
Comment (afc163) [inline README.ar-DZ.md:67]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 25]
```diff
@@ -0,0 +1,135 @@
...
+  - النموذج الأساسي
+  - نموذج الخطوة
+  - نموذج متقدم
+- القائمة
+  - الجدول القياسي
+  - القائمة القياسية
+  - قائمة البطاقات
+  - قائمة البحث (Project/Applications/Article)
+- الملف الشخصي
+  - ملف شخصي بسيط
```
Comment (afc163) [inline README.ar-DZ.md:64]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md (hunk 14 above)
Comment (afc163) [inline README.ar-DZ.md:29]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 26]
```diff
@@ -0,0 +1,135 @@
...
+
+- معاينة: http://preview.pro.ant.design
+- الصفحة الرئيسية: http://pro.ant.design
+- توثيق: http://pro.ant.design/docs/getting-started
+- سجل التغيير: http://pro.ant.design/docs/changelog
+- الأسئلة الشائعة: http://pro.ant.design/docs/faq
+- مرآة الموقع بالصينية: http://ant-design-pro.gitee.io
+
+## 4.0 صدر الآن! 🎉🎉🎉
+
```
Comment (afc163) [inline README.ar-DZ.md:26]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 27]
```diff
@@ -0,0 +1,135 @@
...
+## توظيف الترجمة :loudspeaker:
+
+نحن نحتاج مساعدتك: https://github.com/ant-design/ant-design-pro/issues/120
+
+## الميزات
+
+- :bulb: **تايب سكريبت**: جافا سكريبت لتطوير التطبيقات
+- :scroll: **الكتل**: بناء الصفحة مع قالب كتلة
+- :gem: **تصميم أنيق**: يتبع [مواصفات تصميم النمل](http://ant.design/)
+- :triangular_ruler: **القوالب المشتركة**: قوالب نموذجية لتطبيقات المؤسسة
```
Comment (afc163) [inline README.ar-DZ.md:38]: Codacy found an issue: [[no-heading-punctuation] Don’t add a trailing `:` to headings](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 28]
```diff
@@ -0,0 +1,135 @@
...
+- معاينة: http://preview.pro.ant.design
+- الصفحة الرئيسية: http://pro.ant.design
+- توثيق: http://pro.ant.design/docs/getting-started
+- سجل التغيير: http://pro.ant.design/docs/changelog
+- الأسئلة الشائعة: http://pro.ant.design/docs/faq
+- مرآة الموقع بالصينية: http://ant-design-pro.gitee.io
+
+## 4.0 صدر الآن! 🎉🎉🎉
+
+[الإعلان عن Ant Design Pro 4.0.0](https://medium.com/ant-design/ant-design-pro-v4-is-here-6f23098ae9d9)
```
Comment (afc163) [inline README.ar-DZ.md:27]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

File: README.ar-DZ.md [hunk 29]
```diff
@@ -0,0 +1,135 @@
...
+
+</div>
+
+- معاينة: http://preview.pro.ant.design
+- الصفحة الرئيسية: http://pro.ant.design
+- توثيق: http://pro.ant.design/docs/getting-started
+- سجل التغيير: http://pro.ant.design/docs/changelog
+- الأسئلة الشائعة: http://pro.ant.design/docs/faq
+- مرآة الموقع بالصينية: http://ant-design-pro.gitee.io
+
```
Comment (afc163) [inline README.ar-DZ.md:24]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)

**Pull request merged**